tests/test_get.py

```diff
@@ -0,0 +1,129 @@
+from fastapi.testclient import TestClient
+import pytest
+
+from alwayson.app import app
+from alwayson.controllers.injectors import inject_manifest_repo
+
+
+@pytest.mark.asyncio
+async def test_get_endpoint_single_call(mock_manifest_repo):
+  # Arrange
+  subject = TestClient(app)
+  app.dependency_overrides[inject_manifest_repo] = mock_manifest_repo
+
+  test_manifest = [
+    {
+      'request': {
+        'method': 'GET',
+        'url': 'https://testerozza.com/health'
```

Comment:
Is that domain name open too!? that'd be 🔥

Reply:
That domain is open. I just haven't bought it yet b/c I wasn't sure if I wanted to go with AlwaysOn, Vizier, or Testerozza